auth-web/src/composables/affiliations-factory.ts

```diff
@@ -98,12 +110,17 @@ export const useAffiliations = () => {
         return NrDisplayStates[NrState.HOLD]
       } else return NrDisplayStates[state] || 'Unknown'
     }
-    if (business.status) {
-      return business.status.charAt(0)?.toUpperCase() + business.status?.slice(1)?.toLowerCase()
+    if (business.status === 'HISTORICAL') {
+      return BusinessState.HISTORICAL
```

Comment:
Is this code needed or will line 88 cover it?

Reply:
Updated the logic